Quotient/roommember.cpp

```diff
@@ -133,6 +134,19 @@ QUrl RoomMember::avatarUrl() const {
     return {};
 }
 
+int RoomMember::powerLevel() const
+{
+    if (_room == nullptr || _member == nullptr) {
+        return 0;
```

Comment:
Power levels are allowed to be negative, so I'd rather have:
```suggestion
        return std::numeric_limits<int>::min();
```
to avoid any chance of abuse.